Terminal.Gui/Views/ComboBox.cs

```diff
@@ -22,7 +22,7 @@ public class ComboBox : View {
 		///   Client code can hook up to this event, it is
 		///   raised when the selection has been confirmed.
 		/// </remarks>
-		public event EventHandler<ustring> Changed;
+		public Action<ustring> Changed;
```

Comment:
Let's use this opportunity to name things consistently. In my opinion any event describing when a selected has changed, where the thing changed has an index, the event name should be `SelectedItemChanged`. 

- `ListView`
- `ComboBox`
- `RadioGroup`

Should all be consistent.

Reply:
But wouldn't that break with code already made by users?